webportal/src/app/market_detail/components/copy_popup.jsx

```diff
@@ -110,13 +111,26 @@ export const CopyPopup = props => {
           <DefaultButton
             text='Submit'
             onClick={() => {
+              console.log(marketItem);
+              console.log(copyMarketItem);
               createItem({
                 ...marketItem,
                 ...copyMarketItem,
                 author: user,
                 protocol: JSON.stringify(marketItem.protocol),
-              }).then(() => {
+              }).then(itemId => {
                 hideModal();
+                const qs = queryString.stringify({
+                  itemId: itemId,
+                });
+                history.push(`/market_detail?${qs}`);
```

Comment:
Does the order matter between history.push and update item？

Reply:
It seems to update correctly, but I think it will be better if update the item before history.push